x-pack/plugins/data_visualizer/public/application/common/components/document_count_content/document_count_content.tsx

```diff
@@ -139,6 +118,18 @@ export const DocumentCountContent: FC<Props> = ({
       </div>
     ) : null;
 
+  const CalculatingProbMessage =
```

Comment:
Nit: Suggest to make `ProbabilityUsedMessage` and `CalculatingProbMessage` React components on the outer level and not within that component.

Reply:
Updated here [`1d0e2a6` (#150389)](https://github.com/elastic/kibana/pull/150389/commits/1d0e2a6aa042fd1a8ca2b7973c845d76b8841245)